content/articles/computer-network-services/index.md

```diff
@@ -0,0 +1,194 @@
+
+2. Secures the network from unauthorized users.
+
+3. In a windows network, an active directory is installed on a computer already running a network operating system to make it a server.
+
+4. User and computer accounts are then created in the active directory of the domain controller-users whose accounts are in the active directory and can log onto the network using any computer that is a member of the domain.
+
+- Routing service
+1. This can be used in place of routers.
+
+2. It allows a server with two network interface cards installed to route data between networks or between a network (private) and the internet.
+
+3.  Based on the manner of segmentation, different IP addresses are assigned to the two [network interface cards](https://www.tutorialspoint.com/what-is-network-interface-card-nic) on the network.
+
+### Automation of Network Services
+
+It is an approach to manage network services and resources That is; automating configuration, testing, maintenance, provision, management, and operation of a network and its’ devices both virtual and physical.
+
+This is beneficial to organizations in the following instances;
+
+- Probability of errors is reduced i.e errors due to negligence and oversight from complex network processes is reduced. 
+- Improved operational efficiency. Automation of network reduces task time compared to the manual operation that takes time hence improving efficiency.
+- Operational expenses are reduced. Reduction in operating time, errors, and improved efficiency makes the network fast and agile and as a result, reduces operational costs.
+
+### Network service configurations
+
+This involves the process of assigning network settings to enhance organizational communication over a network. This is achieved through the following steps  by step approaches:
+
+1. Click System then select Network followed by overview for display the configuration menu.
+2. Proceed to the configuration as per the below directives
+3. Save your changes.
+
+| The Settings      |  The configuration Guidelines  |
+| -----------   | ---------|
+| Status        | Show all the interface statistics for all the required ports   |
+| Hostname      | Select a good and descriptive hostname with characters, not exceeding 30.| 
+| Primary DNS   | Indicate primary DNS server IP address.|
+| Secondary DNS | Indicate primary DNS server IP address.|
+| DNS Domain| Indicate a list of default domains. Use commas to separate the domains|
+| Preferred DNS Response| This indicates the preferred requests and responses by the DNS.  Pick ‘V4’ if only IPv4 is needed. Select ‘Both’ if both IPv4 and IPv6 are needed.
+| Bandwidth management | This is used for bandwidth management only.|
+| Maximum Bandwidth| It Specifies the maximum bandwidth for all traffic in the network.|
+| Tunnels Maximum | It Specifies the maximum bandwidth for traffic in the VPN. |
+| Bandwidth | The total maximum bandwidth must be higher than the value of VPN  maximum bandwidth. |
+
```

Comment:
Introduce a section for conclusion. In this section, provide a summary of what the article has covered and the concluding remarks. You can write two paragraphs.